### PR TITLE
docs: fix incorrect resource tag

### PR DESCRIPTION
**Pull request by useradd-temp**
Fixes #N/A 

**Description**

fix incorrect resource tag in `AllowScopedResourceTagging`

If use `"aws:ResourceTag/karpenter.sh/cluster/${ClusterName}": "owned"`, the controller cannot add some tag to the node

**How was this change tested?**

N/A

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [x] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/reference/cloudformation.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ The AllowScopedResourceTagging Sid allows EC2 [CreateTags](https://docs.aws.amaz
   "Action": "ec2:CreateTags",
   "Condition": {
     "StringEquals": {
-      "aws:ResourceTag/karpenter.sh/cluster/${ClusterName}": "owned"
```
Comment (jonathan-innis) [inline website/content/en/docs/reference/cloudformation.md:214]: Good catch 🎉 . Do you mind checking if there are other places where this happened alongside validating that every version of the docs gets this fix (right now this change is just on the `/docs` path)

Comment (useradd-temp) [reply website/content/en/docs/reference/cloudformation.md:214]: I added commit with fix the incorrect tag in another version of the document :) thx for comments

File: website/content/en/preview/reference/cloudformation.md [hunk 2]
```diff
@@ -211,7 +211,7 @@ The AllowScopedResourceTagging Sid allows EC2 [CreateTags](https://docs.aws.amaz
   "Action": "ec2:CreateTags",
   "Condition": {
     "StringEquals": {
-      "aws:ResourceTag/karpenter.sh/cluster/${ClusterName}": "owned"
+      "aws:ResourceTag/kubernetes.io/cluster/${ClusterName}": "owned"
```
Comment (jonathan-innis) [inline website/content/en/preview/reference/cloudformation.md:214]: L205 looks like it needs the same name change here in every file

Comment (useradd-temp) [reply website/content/en/preview/reference/cloudformation.md:214]: Yes, it's the same problem. I fix it

**Review by jonathan-innis — APPROVED** [review]
Nice catch 🎉

**Pull request merged**